README.md

```diff
@@ -12,3 +12,50 @@
+
+Once the project is created, you will be prompted to create or add a Billing Account to your profile.  Once your project has an associated Billing Account, an API key should be generated to authenticate your requests. It can be found under the Credentials section of the APIs & Services tab. Copy this key into your heroku app's Config Vars with the key name `GOOGLE_API_KEY`.  This will allow your app to authenticate and retrieve Google Geocode API results.
+
+#### Getting Zillow API Key
+
+See instructions here [https://www.zillow.com/howto/api/APIOverview.htm](https://www.zillow.com/howto/api/APIOverview.htm)
+
+#### Getting NYC OpenData 311 API Key
+
+
+### Local Project setup
+
+#### MacOS Project Setup
+
+##### Install Python 3
+Although Python 2 is installed by default on most Macs, Python 3 is required for this project. Confirm the installed version of python by opening the Terminal and typing `python --version`. To check to see if Python3 is already installed type `python3 --version` in the terminal and hit enter.  
+
+##### Install Xcode and Homebrew
+We will use the package manager Homebrew to install Python3. Homebrew depends on Apple's Xcode package, so install Xcode through the AppStore or by running the following command in the Terminal.
+`xcode-select --install`
+
+Next, install Homebrew by pasting the following into the terminal and hitting Enter, or by following the instructions here: [http://osxdaily.com/2018/03/07/how-install-homebrew-mac-os/](http://osxdaily.com/2018/03/07/how-install-homebrew-mac-os/):
+`/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`
+
+Finally, install the latest version of Python by running the following command: `brew install python3`
+
+Confirm that it was installed correctly: `python3 --version`. You should see `Python 3.X.X` output in the terminal.
+
+##### Install pip and virtualenv
+
+pip is the preferred way to install dependencies, including Django.  Documentation for installing pip can be found [here](https://pip.pypa.io/en/stable/installing/).  Once pip is installed, it is trivial to install the virtualenv package (for isolating project dependencies).  In the Terminal type `pip install virtualenv`.  
+
+Once virtualenv has been installed, navigate to the cloned project directory and create a local virtualenv folder with `virtualenv venv`. This will create a folder in you project directory called `venv/` which will store your local project dependencies.  Activate you virtual environment by typing `source venv/bin/activate`, and install your local dependencies in the virtual environment by typing `pip install -r requirements.txt`, which will download and install all the project required software listed in the `requirements.txt` file that was cloned with the project. The virtual environment can be turned off anytime by typing `deactivate` in the Terminal in the project base directory.
```

Comment:
It would be great we have some troubleshooting instruction on deploying the project to heroku that you had posted on the slack at the beginning of the semester. I cannot remember the exact commands but you might remember.

```
# not sure
brew install postgres
pip install pycopg2 == 2.7.5
```

Reply:
+100

Reply:
I added the section on the psycopg2 dependency, which is the issue that I had with installing django-heroku.